db/utility.js

```diff
@@ -15,3 +15,62 @@ exports.formatArticles = (articleArray, commentArray) => {
     return newObject;
   });
 };
+
+exports.sortByComments = (articlesArray, order = "desc") => {
```

Comment:
We can do the logic with comments, order by and sort by all using SQL which would be waaay more efficient than JS. 
Have a look at the notes on SQL joins and complex queries again  👀 https://notes.northcoders.com/courses/js-back-end/complex-queries